### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by zebdelrosario**
@EthanSylvester Please check variable/function naming.

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+"""Hex colours program: Enter a colour name and receive its corresponding hex colour."""
+COLOUR_HEX_DICT = {"firebrick": "#b22222", "DodgerBlue1": "#1e90ff", "DeepPink1": "#ff1493", "DarkSlateBlue": "#483d8b",
+                   "DarkOrange": "	#ff8c00", "DarkOliveGreen1": "#caff70", "DarkGoldenrod": "#b8860b",
+                   "coral": "#ff7f50",
+                   "black": "#000000", "AntiqueWhite": "#faebd7"}
```
Comment (EthanSylvester) [inline prac_05/hex_colours.py:5]: Space before code for dark orange can be removed. Black and AntiqueWhite can be moved to previous line.

File: prac_05/word_occurences.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+def main():
+    sentence = input("Text: ")
+    words_occurrences_dict = count_occurrences(sentence)
+    display_dictionary(words_occurrences_dict)
+
+
+def count_occurrences(sentence):
+    """Count how often a word occurs in the given sentence."""
+    words = sentence.split(" ")
```
Comment (EthanSylvester) [inline prac_05/word_occurences.py:12]: " " in .split(" ") is unnecessary as the function automatically uses a space to split if no argument is specified

Comment (EthanSylvester) [inline prac_05/word_occurences.py:12]: Line 12, the " " in .split(" ") is unnecessary as the function automatically does this when no argument is specified

**Review by EthanSylvester — COMMENTED** [review]
Overall the code is very good, just some odd spacing in the hex colours dictionary.

File: prac_05/hex_colours.py (hunk 1 above)
Comment (zebdelrosario) [reply prac_05/hex_colours.py:5]: PEP8 standards suggest not to exceed character count; hence the odd spacing.

File: prac_05/word_occurences.py (hunk 2 above)
Comment (zebdelrosario) [reply prac_05/word_occurences.py:12]: Noted, thank you.

**Pull request merged**